package.json

```diff
@@ -19,7 +19,7 @@
     "lint": "lage lint --continue",
     "lint:copyright": "node scripts/copyrightLinter.js",
     "prepare": "husky install",
-    "change": "beachball change",
+    "change": "beachball change --no-commit",
```

Comment:
Is this intentional?

Reply:
Yes, kind of hate the idea of something automatically commiting for me. Also, not commiting is similar to what we have with `rush` in `itwinjs-core` and `appui`. Any objections?

Reply:
I got used to it commiting automatically for me. 😄 I especially like it in case I forget to run change and CI catches it. As in that case I just need to run `pnpm change` and `git push`.

You can run `pnpm change --no-commit` in those cases when you don't want it to auto commit.

Reply:
ok, I think I can get used to it